test/unit/test_assembly.py

```diff
@@ -139,8 +139,7 @@ def test_lassa_protein(self):
         assembly.order_and_orient(
             os.path.join(inDir, 'contigs.lasv.fasta'),
             os.path.join(inDir, 'ref.lasv.fasta'),
-            outFasta,
-            aligner='promer')
+            outFasta)
```

Comment:
The change to the test is incorrect: it should be aligner=promer. But as you've noted, the current implementation no longer supports this option so the test is commented out. Why don't we leave this as it originally was and leave it skipped for some eventual day when maybe we reimplement the feature.